extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -885,11 +907,12 @@ export class AmpA4A extends AMP.BaseElement {
     // DetachedDomStream.
     streamResponseToWriter(this.win, response, detachedStream);
 
-    this.sanitizedHeadPromise_ = transformStream
+    transformStream
```

Comment:
this is a unreferenced promise chain. might cause issues. do we want to return or catch?

Reply:
or shall we assign it to this.sanitizedHeadPromise_ ? i saw it is used in renderFriendlyTrustless_.

Reply:
Sorry I'm not quite sure what you mean here

Reply:
i don't quite understand the reason for the change in L909 to L912

Reply:
Ah I see, yes the change was so that we initialize as a promise instead of null, otherwise if `layoutCallback` was called before the fetch response returned it would throw that `null` has no `.then` in `renderFriendlyTrustless`.